### PR TITLE
Mark message unread

### LINKED ISSUE #557: [Enhancement]: Ability to mark a read inbox message as unread
### Describe the Current Behavior/Feature:
There is no current functionality to mark an inbox message as unread, once it has been read/clicked on by user.

---

### Proposed Behavior/Feature:
Ability to mark an inbox message as unread again. It should update the unread messages number badge on the message icon in navbar as well.

---

### Rationale:
A lot of users of emails and messages rely on unread message count as a follow-up or priority reminder.

---

### Proposed Implementation (if applicable):
Reverse engineer from the marking an inbox message as read.

---

### Additional Context:
Creating issue (it didn't exist previously) to add more clear visibility/documentation of resulting PR of this addition.

### PR DESCRIPTION
**Pull request by xscottxbrownx**
## This PR:
Resolves #557 
 
**1.**  Adds button to mark inbox message as unread again

## Screenshots (if applicable):

https://github.com/codeforpdx/PASS/assets/71395931/cf7e7735-5f75-44c3-8bef-25fd3536277a

### CODE REVIEW

File: src/components/Messages/MessagePreview.jsx
Comment (leekahung) [inline src/components/Messages/MessagePreview.jsx:1]: Not really an issue, but think we could make the `handleClick` handler function to something like `handleMakeRead` to make it more readable now that we have a `handleMakeUnread` handler.

**Review by leekahung — APPROVED** [review]
Other than the small comment on `handleClick` think it's good to go.

File: src/components/Messages/MessagePreview.jsx
Comment (xscottxbrownx) [reply src/components/Messages/MessagePreview.jsx:1]: updated

**Comment by rioredwards** [comment]
Looks good to me 👍

**Review by rioredwards — APPROVED** [review]

**Pull request merged**